torch/_inductor/autotune_process.py

```diff
@@ -93,19 +97,22 @@ def initialize(self) -> None:
             atexit.register(lambda: self.terminate())
 
         # wait for the initialization to be done
-        self.request_queue.put(Ping())
-        resp = self.response_queue.get()
+        cast("Queue[Any]", self.request_queue).put(Ping())
+        resp = cast("Queue[Any]", self.response_queue).get()
```

Comment:
the proper way to fix this 
```suggestion
        process = self.process
        assert process is not None
        process.start()
```